Source/Directory.Build.targets

```diff
@@ -0,0 +1,10 @@
+<Project>
+  <ItemGroup Condition="$(GeneratePackageOnBuild)">
+    <None Include="$(SolutionDir)..\Icons\OxyPlot_128.png" Pack="true" PackagePath="\" />
+  </ItemGroup>
+  <PropertyGroup Condition="$(GeneratePackageOnBuild)">
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <PackageProjectUrl>https://oxyplot.github.io/</PackageProjectUrl>
+    <PackageIcon>OxyPlot_128.png</PackageIcon>
+  </PropertyGroup>
+</Project>
```

Comment:
The `Build.targets` file is executed towards the end of the build process, so we can depend on variables set in the individual projects. In here we have common properties for our published packages.

Reply:
Typically properties and Items are added to the ]Directory.Build.props] file. ]Directory.Build.targets] is used for custom targets. But in general, it doesn't matter.

You can also add a README for all libraries:
```
  <PropertyGroup Label="Nuget">
    <PackageReadmeFile>README.md</PackageReadmeFile>
  </PropertyGroup>

  <ItemGroup Label="Nuget">
    <None Include="$(MSBuildThisFileDirectory)../../README.md" Pack="true" PackagePath="\" Visible="false" />
  </ItemGroup>
```

I also recommend using $(MSBuildThisFileDirectory) wherever possible instead of $(SolutionDir). $(SolutionDir) is not set when running dotnet build on a specific .csproj file